src/test/java/com/wikia/webdriver/testcases/discussions/CategoriesTests.java

```diff
@@ -0,0 +1,454 @@
+    }
+  }
+
+  @Test(groups = {"discussions-categories-mobile", "discussions-discussionsAdministratorOnMobileCategories"})
+  @Execute(asUser = User.DISCUSSIONS_ADMINISTRATOR)
+  @InBrowser(browser = Browser.CHROME, emulator = Emulator.GOOGLE_NEXUS_5)
+  public void discussionsAdministratorOnMobileCanEditCategoryOnPostsListPage() {
+    final String siteId = Discussions.extractSiteIdFromMediaWikiUsing(driver);
+
+    final PostsListPage page = new PostsListPage().open();
+    final String categoryName = TextGenerator.createUniqueCategoryName();
+
+    CategoriesFieldset categoriesFieldset = page.getFiltersPopOver().click()
+        .getCategoriesFieldset().clickEdit()
+        .rename(EDITABLE_CATEGORY_POSITION, categoryName)
+        .clickApproveButton();
+
+    page.waitForPageReload();
+
+    CategoryPill.Data data = categoriesFieldset.findCategoryWith(categoryName).toData();
+
+    try {
+      Assertion.assertTrue(categoriesFieldset.hasCategory(categoryName), String.format(CATEGORY_SHOULD_BE_VISILBE_IN_LIST_MESSAGE, categoryName));
+      Assertion.assertTrue(isCategoryIn(page.getPostsCreatorMobile(), categoryName), String.format(CATEGORY_SHOULD_BE_VISIBLE_IN_CREATOR_MESSAGE, categoryName));
+    } finally {
+      revertCategoryName(siteId, data);
+    }
+  }
+
+  @Test(groups = {"discussions-categories-mobile", "discussions-discussionsAdministratorOnMobileCategories"})
+  @Execute(asUser = User.DISCUSSIONS_ADMINISTRATOR)
+  @InBrowser(browser = Browser.CHROME, emulator = Emulator.GOOGLE_NEXUS_5)
+  public void discussionsAdministratorOnMobileCanNotAddMoreThanTenCategoriesOnPostsListPage() {
+    final PostsListPage page = new PostsListPage().open();
+
+    CategoriesFieldset categoriesFieldset = page.getFiltersPopOver().click()
+        .getCategoriesFieldset().clickEdit();
+
+    addCategoriesUntilMaxReached(categoriesFieldset);
+
+    Assertion.assertEquals(categoriesFieldset.getInfoMessageText(), CATEGORIES_LIMIT_REACHED_INFO_MESSAGE, INFOR_MESSAGE_SHOULD_APPEAR_MESSAGE);
+  }
+
+  @Test(groups = {"discussions-categories-mobile", "discussions-discussionsAdministratorOnMobileCategories"})
+  @Execute(asUser = User.DISCUSSIONS_ADMINISTRATOR)
+  @InBrowser(browser = Browser.CHROME, emulator = Emulator.GOOGLE_NEXUS_5)
+  public void discussionsAdministratorOnMobileCanRemoveCategoriesOnPostsListPage() {
+    CategoryPill.Data data = DiscussionsCategoryOperations.using(User.DISCUSSIONS_ADMINISTRATOR, driver)
+        .createCategory(TextGenerator.createUniqueCategoryName());
+
+    final String temporaryCategoryName = TextGenerator.createUniqueCategoryName();
+    final PostsListPage page = new PostsListPage().open();
+    final CategoriesFieldset categoriesFieldset = page.getFiltersPopOver().click().getCategoriesFieldset();
+
+    addAndRemoveTemporaryCategory(page, temporaryCategoryName, categoriesFieldset);
+    Assertion.assertNull(categoriesFieldset.findCategoryWith(temporaryCategoryName), "Temporary category should not be added.");
```

Comment:
`TEMPORARY_CATEGORY_SHOULD_NOT_BE_ADDED_MESSAGE` is defined, you can use that here